composer.json

```diff
@@ -52,7 +53,7 @@
         "sort-packages": true
     },
     "conflict": {
-        "ext-redis": "<4.3",
+        "ext-redis": "<5.3",
```

Comment:
Dropped <5.3 because of https://github.com/snc/SncRedisBundle/runs/4280808550?check_suite_focus=true